dlme_airflow/drivers/oai_xml.py

```diff
@@ -59,20 +61,28 @@ def _construct_fields(self, manifest: etree) -> dict:
                         output[field].append(data.text.strip())
         return output
 
-    def uri2label(self, value: str, nsmap: dict):
-        for key in list(nsmap.keys()):
-            if nsmap[key] in value:
-                return value.replace(nsmap[key], "").replace("{}", "")
+    def _get_tag(self, el):
+        """Return the element name, after removing namespace and lowercasing."""
+        return re.sub(r"{.+}", "", el.tag).lower()
 
     # TODO: Discuss if this output shoould be an array (line 63) or a string
     def _from_metadata(self, manifest: etree) -> dict:
         output = {}
-        NS = {"oai_dc": "http://www.openarchives.org/OAI/2.0/oai_dc/"}
-        oai_block = manifest.xpath("//oai_dc:dc", namespaces=NS)[
-            0
-        ]  # we want the first result
+
+        NS = {
+            "oai_dc": "http://www.openarchives.org/OAI/2.0/oai_dc/",
+            "mods": "http://www.loc.gov/mods/v3",
+        }
+
+        if self.metadata_prefix == "oai_dc":
+            oai_block = manifest.xpath("//oai_dc:dc", namespaces=NS)[0]
+        elif self.metadata_prefix == "mods" or self.metadata_prefix == "mods_no_ocr":
+            oai_block = manifest.xpath("//mods:mods", namespaces=NS)[0]
+        else:
+            raise Exception(f"Unknown metadata prefix {self.metadata_prefix}")
```

Comment:
This error is being raised for the `CSIC` collection which is currently configured for `marc21`: https://eu.alma.exlibrisgroup.com/view/oai/34CSIC_INST/request?verb=ListRecords&set=manuscripta&metadataPrefix=marc21

@jacobthill do we want to stick with marc21 for csic or use mods or oai_dc?

Reply:
Actually it doesn't look like the support either:

```<error code="noRecordsMatch">No Publishing profile exists for given set and metadataPrefix</error>```

This appears to be the only collection to not use one of the above known prefixes. 

My inclination is to approve/merge this PR and ticket the need to support marc21 seperately or work with CSIC to support one of the above.